.gitignore

```diff
@@ -0,0 +1,2 @@
+
+*node_modules/
```

Comment:
- [ ] Kindly, make sure that you ignored the `node_modules` folder correctly so that we can't find it remotely in your repo. 
![image](https://user-images.githubusercontent.com/71874425/199477609-74b51986-2eff-4ab1-89ad-966237978fd0.png)

remove the *before it in your `.gitignore` file then run `git rm -r --cached` to remove `node_modules` folder from git control as it was added before.

Reply:
Hello @RolandM99. Thanks for your feedback.  I removed the * sign in the gitignore file. The node module has been removed from the main branch and it is not visible in the "list_structure" branch.  But "git rm -r --cached" is not working for the "list_structure" branch. Could you please advise me? What happens if the files changed stays like this?